src/zenml/services/service.py

```diff
@@ -42,15 +42,55 @@ class ServiceConfig(BaseTypedModel):
         name: name for the service instance
         description: description of the service
         pipeline_name: name of the pipeline that spun up the service
-        pipeline_run_id: ID of the pipeline run that spun up the service
         pipeline_step_name: name of the pipeline step that spun up the service
+        run_name: name of the pipeline run that spun up the service.
     """
 
     name: str = ""
     description: str = ""
     pipeline_name: str = ""
-    pipeline_run_id: str = ""
     pipeline_step_name: str = ""
+    run_name: str = ""
```

Comment:
This change needs to be coupled with a root validator that copies all data from the existing `pipeline_run_id` field into the new `run_name` field, otherwise this information is lost on de-serialization, e.g.:

```
>>> from zenml.services.service import ServiceConfig
>>> d = {"name":"test", "pipeline_run_id": "pipeline_1234"}
>>> ServiceConfig.parse_obj(d)
ServiceConfig(type='zenml.services.service.ServiceConfig', name='test', description='', pipeline_name='', pipeline_step_name='', run_name='')```